pipeline/inputs/windows-event-log-winevtlog.md

```diff
@@ -13,9 +13,9 @@ The plugin supports the following configuration parameters:
 | Interval\_NSec | Set the polling interval for each channel (sub seconds. \(optional\) | 0 |
 | Read\_Existing\_Events | Whether to read existing events from head or tailing events at last on subscribing. \(optional\) | False |
 | DB | Set the path to save the read offsets. \(optional\) |  |
-| String\_Inserts | Whether to include StringInserts in output records. \(optional\) | False  |
+| String\_Inserts | Whether to include StringInserts in output records. \(optional\) | True  |
```

Comment:
I think your head from your branch 1.9 was behind the latest commit (https://github.com/fluent/fluent-bit-docs/commit/bbecdc60291aa454724d80e672a31ecfead34454)
Make sure you always pull all the changes before branching.

Reply:
Updated to lastest changes 
![image](https://user-images.githubusercontent.com/11384973/186030277-f2b82b41-7574-482d-9dd2-d626a96ec667.png)